synapse_invite_checker/invite_checker.py

```diff
@@ -493,6 +505,20 @@ async def on_create_room(
                     errors.Codes.FORBIDDEN,
                 )
 
+        # The room version should always be a string to accommodate arbitrary unstable
+        # room versions. If it was not explicitly requested, the homeserver defaults
+        # will be used. Make sure to check that instance as well
+        room_version: str = request_content.get(
+            "room_version", self.api._hs.config.server.default_room_version.identifier
+        )
+
+        if room_version and room_version not in self.config.allowed_room_versions:
```

Comment:
```suggestion
        if room_version not in self.config.allowed_room_versions:
```

You don't want to allow empty strings as the room version, right?

Reply:
Good catch, I thought I had removed that

fixed in [c34810a](https://github.com/famedly/synapse-invite-checker/pull/41/commits/c34810ae73ecffde5ad73bdc5c93eb06ab4457f7)